pkg/scheduler/core/division_algorithm.go

```diff
@@ -34,10 +34,13 @@ func divideReplicasByResource(
 	spec *workv1alpha2.ResourceBindingSpec,
 	preference policyv1alpha1.ReplicaDivisionPreference,
 ) ([]workv1alpha2.TargetCluster, error) {
-	// Step 1: Get previous total sum of replicas.
-	assignedReplicas := util.GetSumOfReplicas(spec.Clusters)
+	// Step 1: Find the ready clusters that have old replicas
+	scheduledClusters := findOutScheduledCluster(spec.Clusters, clusters)
```

Comment:
What if we just delete the unjoined clusters from `spec.clusters`?

Then the replicas in unjoined clusters will be considered as a special kind of scaling up.

Reply:
Seems we don't need more changes, am I right?

Reply:
it is a  special kind of scaling up